i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+import torch
+from torch import nn
+
+
+class ConformerConvolutionV1(nn.Module):
+    """
+    Conformer convolution module.
+    see also: https://github.com/espnet/espnet/blob/713e784c0815ebba2053131307db5f00af5159ea/espnet/nets/pytorch_backend/conformer/convolution.py#L13
+    """
+
+    def __init__(self, channels: int, kernel_size: int, dropout: float = 0.1, activation: nn.Module = nn.SiLU()):
```

Comment:
`norm` is missing here as an option.

Reply:
Either no default here, and default via the outer module. Or default to batch norm. But not like `norm=nn.BatchNorm()`, because you don't want to reuse the same shared instance.